README.md

```diff
@@ -1 +1,48 @@
-# scriptless-percy
+# Scriptless Percy
```

Comment:
Is this repo a way to generate sitemap file in case not present on the web application by default OR clearly perform certain Percy actions end to end. If #2 is not the case, then the repo name "scriptless-percy" is misleading.

Reply:
This Repo uses sitemap-generator to the get list of URL's from a website and passes those URL's to percy to take snapshots using `percy snapshot` command